harmony/harmony.py

```diff
@@ -552,17 +586,57 @@ def error_messages(self) -> List[str]:
         return error_msgs
 
 
-class LinkType(Enum):
-    """The type of URL to provide when returning links to data.
+class LabelsRequest(BaseRequest):
+    """A Harmony request to create or delete labels for jobs.
+
+    This request allows users to associate labels with Harmony jobs or remove
+    existing labels, facilitating job management and categorization.
 
-    s3: Returns an Amazon Web Services (AWS) S3 URL
-    https: Returns a standard HTTP URL
+    Args:
+        http_method (HttpMethod): The HTTP method to use for the request (e.g., PUT, DELETE).
```

Comment:
I'm willing to be overruled, but I don't like the user specifying the `http_method` here. It is an implementation detail. Also, trying `http_method=HttpMethod.DELETE` gives an error. I would prefer you have separate request types, `AddLablesRequest`, `DeleteLabelsRequest`, but I would settle for using something like `action` instead of `http_method`, where `action` is an enum type defined elsewhere. For now the delete request or delete action would just not be implemented, so no one would be tempted to try deleting yet.

Reply:
I thought about this when implementing. Definitely don't want the action in request name. It's a pain to introduce new Request types. I thought about an `action` attribute, but didn't go that way because it is too loose and indirect, mainly because I don't want to deal with `add`, `create` vs `update` and their meaning on other routes/request types. I can see your point with actually prefer `action` over `http_method`. I can change it to `action` if you feel strongly.

Reply:
Agreed - my first thought also was that `AddLabelsRequest` and `DeleteLabelsRequest` would be better than an `action` parameter.

Reply:
OK. Refactored code.